Integrations/integration-symantec-DLP-incident-reportintg-update.yml

```diff
@@ -0,0 +1,336 @@
+description: Integration to fetch and update symantec DLP Incidents
+detaileddescription: Symantec DLP integration uses SOAP API to communicate with Enforce
+  server to fetch and update  incident Data. uses Zeep package for SOAP Api
+configuration:
+- display: 'credentials '
+  name: credentials
+  defaultvalue: ""
+  type: 9
+  required: false
+- display: 'Server Host (ip or hostname. Specify port followed by ":". ie: dlp-host:8443)'
+  name: server-ip
+  defaultvalue: ""
+  type: 0
+  required: false
+- display: Fetch incidents
+  name: isFetch
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  defaultvalue: ""
+  type: 13
+  required: false
+- display: First fetch timestamp (<number> <time unit>, e.g., 12 hours, 7 days, 3     months,
+    1 year)
+  name: fetch_time
+  defaultvalue: 2 months
+  type: 0
+  required: false
+- display: 'Saved Report ID'
+  name: saved-Report-id
+  defaultvalue: ""
+  type: 0
+  required: false
+script:
+  script: |+
+    from requests import Session
+    from zeep import Client
+    from zeep.transports import Transport
+    from requests.auth import AuthBase, HTTPBasicAuth
+    from zeep import helpers
+    from zeep.cache import SqliteCache
+    import random
+    from datetime import datetime
+    import json
+    import requests
+    from requests.packages.urllib3.exceptions import InsecureRequestWarning
```

Comment:
This import is not used

Reply:
updated the import, was using to disable the ssl warnings